src/task_manager/robosub_state_machine_node.py

```diff
@@ -0,0 +1,47 @@
+import roslib
+
+import rospy
+import smach
+import smach_ros
+
+# states
+from sensor_checker_state import Sensor_Checks
+from dive_state import Dive_State
+
+from actionlib import *
+from actionlib_msgs.msg import *
+
+def main():
+    rospy.init_node('Robosub_StateMachine')
+
+    sm_top = smach.StateMachine(outcomes=['Mission_Completed', 'Mission_Failed'])
```

Comment:
How does this get logged? Would adding more failure states be useful to see what went wrong during a run?

Reply:
Yes we can add more failures states. But there has to something using the state. For now I will add the sensor check failure and later will add more.

Reply:
:+1: